hack/last-fed-sha

```diff
@@ -0,0 +1 @@
+40ab598d06835906b0e039d1f92a5f10ef079759
```

Comment:
better to keep these files along with the script in some specific directory within hack directory. In future we may need to store additional files/scripts under hack, which may get mixed up.